assets/chezmoi.io/docs/reference/special-files-and-directories/chezmoiexternal-format.md

```diff
@@ -35,6 +35,13 @@ Entries may have the following fields:
 | `refreshPeriod`   | duration | `0`           | Refresh period                                                |
 | `stripComponents` | int      | `0`           | Number of leading directory components to strip from archives |
 | `url`             | string   | *none*        | URL                                                           |
+| `checksum.sha256` | string   | *none*        | Expected SHA256 checksum of data                              |
+| `checksum.sha384` | string   | *none*        | Expected SHA384 checksum of data                              |
+| `checksum.sha512` | string   | *none*        | Expected SHA512 checksum of data                              |
```

Comment:
I would *personally* document `` `checksum.size` `` with the caveat `Expected number of bytes of data, should not be used by itself`.

The benefit of `checksum.size` is that it’s a good cross-check against size stuffing to get a collision.

Reply:
I've added `checksum.size` to the table.